scripts/copy-skia-module-headers.ts

```diff
@@ -13,8 +13,7 @@ const copyModule = (module: string) => [
   `cp -a ./externals/skia/modules/skcms/. ./package/cpp/skia/modules/skcms`,
   `mkdir -p ./package/cpp/skia/src/`,
   `mkdir -p ./package/cpp/skia/src/core/`,
-  `cp -a ./externals/skia/src/core/SkLRUCache.h ./package/cpp/skia/src/core/.`,
-  `cp -a ./externals/skia/src/core/SkTInternalLList.h ./package/cpp/skia/src/core/.`,
+  `cp -a ./externals/skia/src/core/SkTHash.h ./package/cpp/skia/src/core/.`,
```

Comment:
here I first removed the SkLRUCache.h SkTInternalLList.h and then added the ones which were needed. This is sensible to do this for every upgrade since these private API will be changing a lot everytime.